ps1_pv3_3pi_20170110/README.txt

```diff
@@ -0,0 +1,487 @@
+PSO J000134.268+451710.163  162340003927854012  00 01 34.272    +45 17 10.03    0.015   0.016   53  16836277578733  16234   0.825292547070694   2225    34  3   444915712   53  00 01 34.269    +45 17 10.16    0.001   0.001   55936.29421296  1.457   22.19887281378932   40.54697590262472   113.808085852593    -16.70034366172768  -999    5   4   12  13  12  12  0.999   21.985  0.026   0.055   3   21.927  22.035  21.543  0.132   0.492   4   21.459  0.240   0.527   4   16892216    0.999   20.877  0.067   0.238   11  20.602  21.424  20.644  0.059   0.200   11  20.608  0.084   0.197   11  16892216    0.999   20.302  0.022   0.196   11  19.954  20.652  19.861  0.027   0.131   8   19.825  0.028   0.112   8   16892216    0.999   19.820  0.042   0.131   10  19.605  19.969  19.496  0.021   0.157   9   19.470  0.029   0.136   10  16892216    1.000   19.499  0.037   0.149   10  19.304  19.851  19.122  0.054   0.168   10  19.058  0.049   0.170   10  16892216    3.467e-7
+
+The magnitudes appear to be identical (within rounding)!  The quoted errors are
+slightly different --- no big deal.
+
+
+Recompress everything with gzip
+-------------------------------
+
+A warning about the DVO files, from http://dvodist.ipp.ifa.hawaii.edu :
+
+       Note that all but the first 4 DVOs are distributed with compressed
+       FITS tables following the convention specified by Pense et al
+       (http://arxiv.org/pdf/1201.1340v1.pdf). This convention has not yet
+       been ratified by the FITS standard, though is expected by Summer
+       2016. As a result, standard FITS tools will have trouble reading 
+       these tables. Two options which exist: 1) download the current IPP
+       / Ohana tools and use these either to read the tables (with e.g.,
+       dvo or mana), or to uncompress the tables (with the program  
+       ftable). 2) download the latest version of fpack and use funpack to
+       uncompress the tables. In the later case, there is a small
+       inconsistency between our implementation and funpack: if funpack  
+       finds the header keyword ZTILELEN and the expected output table   
+       length ZNAXIS2 is < ZTILELEN, funpack will fail. (We treat ZTILELEN
+       as a maximum block size, and so handle this case as we handle the
+       last block in a table. The white paper is not explicit in how to
+       handle this case). It is possible to use funpack to uncompress  
+       these tables by modifying the value of ZTILELEN to match ZNAXIS2.
+
+That's rather annoying: the tool most everyone has available doesn't work
+out-of-the-box, and the tool that does work is a bit obscure. In order to
+make this more convenient, decompress everything and recompress it with
+gzip.  That way, cfitsio and pyfits will just work without having to do
+anything special. And I think it saves a bit of space as well (maybe 25%?).
```

Comment:
Worth noting here that recompressing the whole file in gzip does result in a performance hit (the whole file has to be decompressed to read any of it, including the header)?